### PR TITLE
Feature/endpoint count of sessions

### PR DESCRIPTION
**Pull request by KoR82**

### CODE REVIEW

File: backend/app/Http/Resources/CountSessions.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+use Illuminate\Http\Resources\Json\JsonResource;
+use App\Contracts\ApiResponse;
+
+class CountSessions extends JsonResource implements ApiResponse
+{
+    public function toArray($request): array
+    {
+        return [
+            'qty_sessions' => $this->resource
```
Comment (lenchvolodymyr) [inline backend/app/Http/Resources/CountSessions.php]: What is qty?

File: backend/tests/Feature/CountSessionsTest.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        $browserId = factory(\App\Entities\Browser::class)->create()->id;
+        $pageId = factory(\App\Entities\Page::class)->create(['website_id' => $website->id])->id;
+        $demographicId = factory(\App\Entities\Demographic::class)->create(['geo_position_id' => $geoPositionId])->id;
+        $deviceId = factory(\App\Entities\Device::class)->create()->id;
+        $systemId = factory(\App\Entities\System::class)->create([
+            'browser_id' => $browserId,
+            'os_id' => $oSId
+        ])->id;
+
+        factory(Session::class, 30)->create([
```
Comment (lenchvolodymyr) [inline backend/tests/Feature/CountSessionsTest.php]: The same problem as in the average session pull request

File: backend/app/Http/Resources/CountSessions.php (hunk 1 above)
Comment (KoR82) [reply backend/app/Http/Resources/CountSessions.php]: qty is short form of quantity ) 
need to fix it?

Comment (lenchvolodymyr) [reply backend/app/Http/Resources/CountSessions.php]: yes, don't use acronyms and short forms, unless it is obvious)

**Pull request merged**